src/components/atoms/ProfileIcon/ProfileIcon.tsx

```diff
@@ -2,8 +2,8 @@ import React, { ComponentPropsWithRef, ForwardedRef, forwardRef } from 'react';
 import { NormalProfile } from '@/assets';
 import { profileWrapper, profileImage } from './ProfileIcon.style';
 
-interface ProfileProps extends ComponentPropsWithRef<'button'> {
-  interaction: 'normal';
+export interface ProfileProps extends ComponentPropsWithRef<'button'> {
+  interaction: 'normal' | 'settings';
```

Comment:
```suggestion
  interaction: 'default' | 'custom';
```
으로 네이밍이 변경되어도 좋아보여요! 기본 프로필 이미지 / imgUrl이 설정되어 있는 커스텀 프로필 이미지 라는 뜻에 부합하는 거 같습니다